stargazer/stargazer.py

```diff
@@ -176,6 +178,14 @@ def reset_covariate_order(self):
         if self.original_cov_names is not None:
             self.cov_names = self.original_cov_names
 
+    def add_custom_lines(self, lines, location='footer top'):
+        for line in lines:
+            assert len(line) == self.num_models + 1, \
+                'Please input iterables of length "number of models + 1"'
+        assert location in self.CUSTOM_LINES_LOCATIONS,  \
+            'location needs to be one of {}'.format(str(self.CUSTOM_LINES_LOCATIONS))
+        self.custom_lines[location] = lines
```

Comment:
This overwrites the current value, so only one line can be added per location, right?

It'd be nice to be able to add multiple lines per location, such that this would be some sort of append statement.

Reply:
> It'd be nice to be able to add multiple lines per location, such that this would be some sort of append statement.

Good point: it would be great if each ``self.custom_lines[location]`` was a ``list``, and ``add_line`` appended lines to it. Then (at least for now) we could just mention in the docstrings that one can remove all added lines with ``table.custom_lines[location] = []``.